README.md

```diff
@@ -27,13 +53,16 @@ set_line(buf, 1, "some text")
 
 A high level interface is work in progress. For the moment `Buffer` supports simple array operations, please see `test/runtests.jl` for examples.
 
-The module exports a low-level interface for handling asynchronous events (notifications and requests). A prototype (read: ugly hack) implementation of vim bindings for the julia REPL is included as an example, see `src/repl.jl`.
+The module exports a low-level interface for handling asynchronous events (notifications and requests). A prototype (read: ugly hack) implementation of Vim bindings for the Julia REPL is included as an example, see `src/repl.jl`.
 
-This package also includes a remote plugin host, similar to the one in python-client. To use it, it is recommended to manage this package using the Julia package manager (as it handles dependencies and julia package path), and also add this repo root to runtimepath in nvimrc:
 
-    set rtp+=~/.julia/Neovim/
+### Using as a plugin host
 
-A julia plugin can then be defined in a `rplugin/julia/` subfolder to your nvim folder or to a plugin repo. Functions defined at toplevel can be exported using macros in the Neovim module. `@fn`, `@command`, `@autocmd` can be used, as well as variants ending with `sync`.
+This package also includes a remote plugin host, similar to the one in the Python client [pynvim](https://github.com/neovim/pynvim). To use it, add this repo root to `runtimepath` in `init.vim`:
+```
+set rtp+=~/.julia/packages/Neovim/
+```
+A Julia plugin can then be defined in a `rplugin/julia/` subdirectory to your nvim directory or to a plugin repo. Functions defined at toplevel can be exported using macros in the Neovim module. `@fn`, `@command`, `@autocmd` can be used, as well as variants ending with `sync`.
```

Comment:
May want to add a parenthetical about what "your nvim directory" usually is (`$XDG_CONFIG_HOME/.config/nvim`)?

Also, may also want to add some text there at the end like, "Here's an example:" so that it flows better into the example code underneath.